### ISSUE TITLE
Can PLLaVA caption rectangular videos without cropping?

### ISSUE THREAD

**Issue by tomyoung903** [magic-research/PLLaVA#38]
I notice that the image processor crops rectangular images into square images, which inevitably loses some information.

It seems that cropping is also used during training.

What if we want to caption rectangular videos without losing edges to cropping?

**Comment by ermu2001** [comment]
Yes, we've conducted conducted all our experiments with cropping.

Padding would work regarding retaining all information, but would downgrade the resolution and doesn't aligns to the training strategy of image models (CLIP and llava-next).
